_data/projects/MERN_CRUD_Online_store.yml

```diff
@@ -0,0 +1,18 @@
+name: Online store app with MERN 
+desc: A boilerplate for a front and and back end API. An online store app is instantiated.
+site: https://github.com/aawgit/onlineStore
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- MERN
+- JavaScript
+- NodeJS
+- ReactJS
+- MongoDB
+- Mongoose
+- Express
+- Cloudinary
+- Passport-facebook
+- Oauth
+upforgrabs:
+  name: good first issue
+  link: https://github.com/aawgit/onlineStore/blob/master/README.md
```

Comment:
@aawgit link should be https://github.com/aawgit/onlineStore/labels/good%20first%20issue

Reply:
Thanks. Changes are done.